src/Nethermind/Nethermind.BeaconNode.Test/MockedStart/QuickStartTest.cs

```diff
@@ -145,6 +146,97 @@ public async Task TestValidatorKeyGeneration10()
             }
         }
 
+        [TestMethod]
+        public async Task TestValidatorKeyGeneration64()
+        {
+            // Arrange
+            IServiceCollection testServiceCollection = TestSystem.BuildTestServiceCollection(useStore: true);
+            IConfigurationRoot configuration = new ConfigurationBuilder()
+                .AddInMemoryCollection(new Dictionary<string, string> {["QuickStart:ValidatorCount"] = "64"})
+                .Build();
+            testServiceCollection.AddQuickStart(configuration);
+            ServiceProvider testServiceProvider = testServiceCollection.BuildServiceProvider();
+
+            // Act
+            INodeStart quickStart = testServiceProvider.GetService<INodeStart>();
+            await quickStart.InitializeNodeAsync();
+
+            // Assert
+            IStoreProvider storeProvider = testServiceProvider.GetService<IStoreProvider>();
+            storeProvider.TryGetStore(out IStore? store).ShouldBeTrue();
+            if (!store!.TryGetBlockState(store.FinalizedCheckpoint.Root, out BeaconState? state))
+            {
+                throw new InvalidDataException("Missing finalized checkpoint block state");
+            }
+            state!.Validators.Count.ShouldBe(64);
+        }
+
+        
+//        [TestMethod]
```

Comment:
ideally always add a comment explaining why the thing is commented out and when it can be removed
e.g. tks: commenting out until the x is fixed, can be removed if not commented out by 01/03/2020